pages/learn/manage/device-statuses.md

```diff
@@ -5,35 +5,44 @@ excerpt: Determine the status and connectivity of a device
+| **Post Provisioning**      | The device has been [provisioned][device-provisioning] but has not yet come online.                                                                             |
+| **Inactive**               | The device has been [deactivated][deactivated] or has been [preregistered][preregistered] but has not yet connected to the {{ $names.cloud.lower }} API.        |
+| **Frozen**                 | The device has been frozen because it's outside the organization's allowance, or is in a paid [application type][application type] on a free tier organization. |
 
-| Status            | Description                                                                                                                                                     |
-|-------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| Online            | The device is online and is connected to the VPN or has recent API communications.                                                                              |
-| Configuring       | The device is applying OS configuration.                                                                                                                        |
-| Updating          | The device is updating to a new application release.                                                                                                            |
-| Offline           | The device is offline and is not connected to VPN and has not any recent API communications.                                                                    |
-| Post Provisioning | The device has been [provisioned][device-provisioning] but has not yet come online.                                                                             |
-| Inactive          | The device has been [deactivated][deactivated] or has been [preregistered][preregistered] but has not yet connected to the {{ $names.cloud.lower }} API.        |
-| Frozen            | The device has been frozen because it's outside the organization's allowance, or is in a paid [application type][application type] on a free tier organization. |
-
-## Device connectivity
+## Device Connectivty states
 
 ![Device connectivity indicators](/img/common/main_dashboard/device_status.png)
 
 In addition to the device status, there are indicators to show if the device is having any partial connectivity issue. These indicators can be used to identify common issues, such as a firewall blocking VPN traffic or an inability of the device to communicate with the {{ $names.cloud.lower }} API.
 
-For example, in case a device is connected to the API only, indicated by a status of Online (Heartbeat Only), you may still see logs being sent. This case indicates that the device is unable to connect to the {{ $names.cloud.lower }} VPN.
+Some statuses may indicate partial connectivity issues:
```

Comment:
Remove this line if using the suggestion above that ends with "These indicators are as follows:"

Reply:
I'm going to go ahead and force-push this change - looks like it was prematurely resolved